src/api/schemas.py

```
@@ -0,0 +1,278 @@
+"""
+Pydantic schemas for the Fraud Detection API.
+
+Defines request and response models with validation.
+"""
+
+from pydantic import BaseModel, Field, validator
+from typing import Dict, Any, List, Optional, Union
+from datetime import datetime
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)